nservicebus/transactional-session/index_remote-processor_transactionalsession_[3.3,).partial.md

```diff
@@ -0,0 +1,44 @@
+## Send only 
+
+If used in a [send-only](/nservicebus/hosting/#self-hosting-send-only-hosting) endpoint the transactional session must be configured with a remote endpoint that will manage the outbox on behalf of the endpoint. This processor endpoint must have both the outbox and the transactional session enabled.
+
+snippet: configure-remote-processor
+
+> [!WARNING]
+> Both the send only endpoint and the processor endpoint must be connected to the same database. See [documentation for the individual persisters](/persistence/) for more details.
+
+> [!NOTE]
+> If migrating to this mode from an endpoint couldn't be made send only due to the previous limitations of the transactional session, see the migration guidance below.
+
+### Outbox cleanup
+
+For persisters where [Outbox cleanup](/nservicebus/outbox/#outbox-expiration-duration) is performed by the endpoint instances, only the remote processing endpoint will have the cleanup enabled to prevent concurrent cleanup from happening.
+
+Applies to:
+
+- SQL Persistence
+- NHibernate
+
+### Migration to send only endpoint mode
+
+For endpoints that couldn't be send only due the limitations of the transactional session use the following procedure to migrate your endpoint.
+
+#### Preparation
+
+1. Ensure that no message handlers exists in the endpoint
+1. Deploy a new processor endpoint
```

Comment:
See in this same file, we have a reference to a partial called `partial: remote-processor`:
- Where is that snippet?
- The concept of a remote processor needs to be clearly established before we reference the concept